src/system/src/service/cli/cmd/help.rs

```diff
@@ -0,0 +1,17 @@
+//
+//  help.rs
+//  GunwOS
+//
+//  Created by Artur Danielewski on 09.01.2023.
+//
+
+#![no_std]
+
+extern "C" { pub fn s_trm_puts(szText: &str); }
```

Comment:
```suggestion
pub extern "C" fn s_trm_puts(szText: &str);
```

Reply:
Sprawdziłem:

error[E0449]: unnecessary visibility qualifier
  --> ./src/service/cli/cmd/help.rs:10:1
   |
10 | pub extern "C" { fn s_trm_puts(szText: &str); }
   | ^^^ `pub` not permitted here because it's implied
   |
   = note: place qualifiers on individual foreign items instead

Reply:
Ale można usunąć pub całkiem i też działa